.gitignore

```diff
@@ -42,3 +42,4 @@
 
 # Ignore Spec Cache
 spec/examples.txt
+vendor
```

Comment:
Can you tell us more about where this `vendor` folder comes from? 👀

Reply:
It is generated when I type `bundle install`

Reply:
ok, maybe from one of the gem specific to auth that I haven't used yet 🤔 
Thanks ~